src/utils/sanitizeUrlHelper.js

```diff
@@ -0,0 +1,9 @@
+import { sanitizeUrl } from '@braintree/sanitize-url';
+
+const sanitizeUrlHelper = (url) => {
+  const sanitizedUrl = sanitizeUrl(url);
+  const checkerStartHttp = ['http', 'https'].some((element) => sanitizedUrl.includes(element));
```

Comment:
```suggestion
  const checkerStartHttp = ['http://', 'https://'].some((element) => sanitizedUrl.startsWith(element));
```

Otherwise it would allow urls like `http.im.evil.com` and `www.im.evil/http`.